### ISSUE TITLE
RemoteStartTransaction gets rejected.

### ISSUE THREAD

**Issue by mincequi** [goecharger/go-eCharger-API-v2#243]
Hi there,

i have a go-eCharger V4 with this Firmware 56.9.
However, the following RemoteStartTransaction gets rejected:

[2,"RemoteStartTransaction 31","RemoteStartTransaction",{"chargingProfile":{"chargingProfileId":1,"chargingProfileKind":"Absolute","chargingProfilePurpose":"TxProfile","chargingSchedule":{"chargingRateUnit":"A","chargingSchedulePeriod":[{"limit":6,"numberPhases":1,"startPeriod":0}]},"stackLevel":0},"idTag":"user"}]

Do you have any idea, what could be wrong here?

Thanks a lot.

**Comment by 0xFEEDC0DE64** [comment]
I dont think this is valid OCPP at all, why mixing RemoteStartTransaction with a charging profile? Does the spec allow this?

**Comment by mincequi** [comment]
According to this spec: https://groups.oasis-open.org/higherlogic/ws/public/download/58944/ocpp-1.6.pdf/latest

**Yes**, that should work.

or: which way would you suggest to continuously adapt the allowed charging power/amps for PV surplus charging?

**Comment by 0xFEEDC0DE64** [comment]
gotcha! I havent seen RemoteStartTransaction mixed with a chargingProfile yet, we currently only support SetChargingProfile and ClearChargingProfile.

I will put your example on my todo list

**Comment by EnvironCMO** [comment]
Does anyone have a working example of a SetChargingProfile command? My commands are accepted but unfortunately the wallbox does not execute these commands
